Example/SmileID/BiometricKYC/BiometricKycWithIdInputScreen.swift

```diff
@@ -9,75 +9,81 @@ struct BiometricKycWithIdInputScreen: View {
     @StateObject var viewModel: BiometricKycWithIdInputScreenViewModel
 
     var body: some View {
-        switch viewModel.step {
-        case let .loading(messageKey):
-            VStack {
-                ActivityIndicator(isAnimating: true).padding()
-                Text(SmileIDResourcesHelper.localizedString(for: messageKey))
-                    .font(SmileID.theme.body)
-                    .foregroundColor(SmileID.theme.onLight)
-            }
-            .frame(maxWidth: .infinity)
-        case let .idTypeSelection(countryList):
-            VStack {
-                SearchableDropdownSelector(
-                    items: countryList,
-                    selectedItem: selectedCountry,
-                    itemDisplayName: { $0.name },
-                    onItemSelected: { selectedCountry = $0 }
-                )
-                if let selectedCountry = selectedCountry {
-                    RadioGroupSelector(
-                        title: "Select ID Type",
-                        items: selectedCountry.availableIdTypes,
-                        itemDisplayName: { $0.label },
-                        onItemSelected: { idType in
-                            viewModel.onIdTypeSelected(
-                                country: selectedCountry.countryCode,
-                                idType: idType.idTypeKey,
-                                requiredFields: idType.requiredFields ?? []
-                            )
-                        }
+        ZStack {
+            switch viewModel.step {
+            case let .loading(messageKey):
+                VStack {
+                    ActivityIndicator(isAnimating: true).padding()
+                    Text(SmileIDResourcesHelper.localizedString(for: messageKey))
+                        .font(SmileID.theme.body)
+                        .foregroundColor(SmileID.theme.onLight)
```

Comment:
should we replace this with the .light option or this does not get affected by dark mode on the device

Reply:
This is from the theme class. Not affected by dark mode on the device. 
PS: This is the same as on the main branch. I only wrapped the body of the screen inside a ZStack to prevent glitches.